GCRCatalogs/utils.py

```diff
@@ -32,3 +33,13 @@ def first(iterable, default=None):
     returns the first element of `iterable`
     """
     return next(iter(iterable), default)
+
+
+def decode(bytestring):
+    """
+    decode a bytestring is possible
```

Comment:
Suggest  docstring text include the justification as in the pull request, e.g. something like

decode presumed bytestring.  This utility smooths over differences in return values between, in particular, v2 and v3 of h5py

Reply:
Thanks! I expanded the docstring!